decko-rails/lib/decko/rails.rb

```diff
@@ -1,11 +1,33 @@
 DECKO_RAILS_GEM_ROOT = File.expand_path('../../..', __FILE__)
 
+require 'rails/all'
+require 'decko/engine'
+
 module Decko
-  module Rails
+  module Rails # not sure we need this
     class << self
       def gem_root
         DECKO_RAILS_GEM_ROOT
       end
     end
   end
-end
\ No newline at end of file
+
+  if defined? ::Rails::Railtie
```

Comment:
I think this file won't even be loaded in the regular wagn app case.  This file has to trigger any glue logic that is needed for the engine deck case.  In that case, I think this better be defined by now or it won't work, so maybe it isn't even needed.